configure.ac

```diff
@@ -4,7 +4,7 @@ AC_CONFIG_HEADER([src/config.h])
 ## Use xapian-config-1.3 to find arguments for compiler and linker flags
 ##
 ## Check for non-standard programs: xapian-config-1.3
-AC_PATH_PROG([XAPIAN_CONFIG], [xapian-config-1.3])
+AC_PATH_PROG([XAPIAN_CONFIG], [xapian-config], [xapian-config-1.3])
```

Comment:
You'll need to rebuild `configure` and commit that for this change to be picked up (and hence the travis tests to pass).